tests/x11regressions/gnomecase/application_starts_on_login.pm

```diff
@@ -52,13 +40,20 @@ sub start_dconf {
 sub alter_status_auto_save_session {
     my ($self) = @_;
     $self->start_dconf;
-    send_key_until_needlematch "dconf-org", "down";
-    assert_and_click "unfold";
-    send_key_until_needlematch "dconf-org-gnome", "down";
-    assert_and_click "unfold";
-    send_key_until_needlematch "dconf-gnome-evolution", "down";
-    assert_and_click "scroll-down";    #this step aim to work around screen not scroll down automate issue
-    send_key_until_needlematch "gnome-session", "down";
+    if (sle_version_at_least('15')) {
```

Comment:
if this behaviour is true for SLE15 and more recent wouldn't it also apply for TW?

Reply:
I found that sle_version_at_least('15') also apply for TW, but I haven't looked into the testapi.

Reply:
yes, of course, I forgot.

Reply:
fixed: http://10.67.17.30/tests/1922